RequestResponse.swift

```diff
@@ -14,10 +14,12 @@ public class RequestResponse {
         var request = URLRequest(url: url)
         request.setValue("application/json", forHTTPHeaderField: "Content-Type")
         request.httpMethod = "POST"
-        
+
+        let encoder = JSONEncoder()
+        encoder.keyEncodingStrategy = .convertToSnakeCase
```

Comment:
You were _decoding_ by converting from snake case, but not encoding _to_ snake case.

Reply:
Hmmm. But the encoding is working. The decoding isn't working. When you hit "save" the row gets wrote (written?) to the DB. It's the decode of the response data that blows up. That's why I have the underscore in the "user_id" property that's the name of the column in the DB and what comes back in the response data:
{\"id\":1,\"name\":\"ST\",\"description\":\"ST\",\"ispublic\":true,\"user_id\":1}

Reply:
I shouldn't have been doing the "convertFromSnakeCase" on the decoder. Removing that fixed it.